src/document/DocumentCommandHandlers.js

```diff
@@ -159,8 +159,11 @@ define(function (require, exports, module) {
             windowTitle         = brackets.config.app_title,
             currentlyViewedFile = MainViewManager.getCurrentlyViewedFile(MainViewManager.ACTIVE_PANE),
             currentlyViewedPath = currentlyViewedFile.fullPath,
-            readOnlyString      = currentlyViewedFile.readOnly ? "[Read Only] - " : "";
-
+            readOnlyString      = currentlyViewedFile.readOnly ? "[Read Only] - " : "",
+            filepathDisplayPref = PreferencesManager.get('showFullPathToFile');
+            // Toggles between display of full or truncated filepath in top bar based on setting preference
+            var pathToFile = filepathDisplayPref === "false" ? _currentTitlePath : currentlyViewedPath;
```

Comment:
Can you change this tertiary operator from explicitly checking for "false" to something like
`var pathToFile = filepathDisplayPref ? currentlyViewedPath : _currentTitlePath;`

Existing check might give incorrect result.

Reply:
Thanks again. I've now made this change too.